app/src/main/java/com/android/solvit/shared/ui/chat/ChatScreen.kt

```diff
@@ -75,6 +98,11 @@ fun ChatScreen(
   val user by authViewModel.user.collectAsState()
   chatAssistantViewModel.setContext(messages, "Hassan", receiverName)
```

Comment:
Is this a hard coded value ?

Reply:
Yes, but it's someone else who added this line in the file. But I will solve it in another issue